spec/models/item_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Item, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:item) { create(:item, name: "Bacon Maple Crunch",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.